bip-taproot.mediawiki

```diff
@@ -209,7 +209,7 @@ The following function, <code>taproot_output_script</code>, returns a byte array
 def taproot_tree_helper(script_tree):
     if isinstance(script_tree, tuple):
         leaf_version, script = script_tree
-        h = tagged_hash("TapLeaf", bytes([leaf_version]) + ser_script(script))
+        h = tagged_hash("TapLeaf", bytes([(2 * leaf_version) ^ 0xc0]) + ser_script(script))
```

Comment:
Need to also shift and xor the leaf version in `taproot_sign_script`

Reply:
Done.

Reply:
Done.